app.js

```diff
@@ -4,16 +4,22 @@ const cors = require('cors')
 require('dotenv').config()
 const setJWTStrategy = require('./config/jwt')
 const authMiddleware = require ('./middleware/jwt.js')
-
 const contactsRouter = require('./routes/api/contacts')
 const authRouter = require('./routes/api/authRouter')
 
+const path = require('path')
+
 const app = express()
 
 const corsOptions = {
   origin:[`https://www.google.pl`]
 }
 
+
+
```

Comment:
Staraj się mieć nie wiecej niz jedna linie pusta horyzontalnie. To ładniej wygląda.